services/catalog/service/src/common/constants.ts

```diff
@@ -1 +1,9 @@
 export const DEFAULT_LOCALE_TAG = 'default';
+/**
+ * The HTTP Request header key for locale filtering.
+ */
+export const MOSAIC_LOCALE_HEADER_KEY = 'mosaic-locale';
```

Comment:
shouldn't this be something like `X-`?

did we think about using the `Accept-Language` header?

Reply:
"Custom proprietary headers have historically been used with an X- prefix, but this convention was deprecated in June 2012" (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers)

Reply:
ok, not an `X-` then :-)

About GraqphQL over HTTP - we can do that, once it's getting somewhere.

Question for now would be, should we use the `Accept-Language` header? Maybe we could use our header as preferred, and if it's not present, we try to find a locale for the `Accept-Language`?

Reply:
For simplicity's sake, I would stick with only a custom header. Since this is a customizable service - this can be adjusted based on the needs of the custom frontends.